lib/dotcom/content_rewriters/links.ex

```diff
@@ -49,7 +49,7 @@ defmodule Dotcom.ContentRewriters.Links do
     href = Map.get(attr_map, "href", "")
```

Comment:
Looks like when it's a list it's overwhelmingly a single string in the list. I think we should use this string instead of ignoring it with your change below. Maybe something

```elixir
href = case Map.get(attr_map, "href") do
  href when is_binary(href) -> href
  [href] -> href
  _ -> ""
end
```